editor/editor_export.cpp

```diff
@@ -1803,6 +1803,41 @@ bool EditorExportPlatformPC::can_export(const Ref<EditorExportPreset> &p_preset,
 	valid = dvalid || rvalid;
 	r_missing_templates = !valid;
 
+#ifdef WINDOWS_ENABLED
```

Comment:
rcedit can be used under Linux and macOS with wine, so this code should not be specific to Windows editor binaries.

Reply:
I replaced macro with `get_os_name` and I hope this one can work.